docs/dashboards/types-of-components.md

```diff
@@ -12,37 +11,38 @@ This is an overview of the most critical parameters of a component:
 * `events` - an object containing a pair of names for the event and a callback function that should be called on a given event. The list of events can be found in the API Reference, but the most common is `mount`.
 * `sync` - list of events that should be synchronized between components.
 
-### HTML Component
+## HTML Component
 The most basic and generic component type allows you to add everything that could be defined as HTML, as well as some custom events.
 
 The HTML component can be defined in two ways:  
-1. **Abstract Syntax Tree (AST) style**  
-    Defines the HTML code as a nested tree structure.
-
-    Code snippet:
-    ```js
-    {
-        type: 'HTML',
-        renderTo: 'dashboard-1',
-        elements: [{
-            tagName: 'img',
-            attributes: {
-                src: 'https://www.highcharts.com/samples/graphics/stock-dark.svg'
-            }
-        }]
-    }
-    ```
-2. **HTML string**  
-    Defines the HTML code as a standard HTML string.
-
-    Code snippet:
-    ```js
-    {
-        type: 'HTML',
-        renderTo: 'dashboard-1',
-        html: '<img src="https://www.highcharts.com/samples/graphics/stock-dark.svg">'
-    }
-    ```
+### **Abstract Syntax Tree (AST) style**  
```

Comment:
On a general basis we shouldn't use ** in #. Let the CSS handle formatting of headers.